tests/components/hue/test_init_multiple_bridges.py

```diff
@@ -0,0 +1,185 @@
+        assert (
+            await async_setup_component(
+                hass,
+                hue.DOMAIN,
+                {},
+            )
+            is True
+        )
+
+
+async def test_hue_activate_scene_both_responds(
+    hass, mock_bridge1, mock_bridge2, mock_config_entry1, mock_config_entry2
+):
+    """Test that makes both bridges successfully activate a scene."""
+
+    await setup_component(hass)
+
+    await setup_bridge(hass, mock_bridge1, mock_config_entry1)
+    await setup_bridge(hass, mock_bridge2, mock_config_entry2)
+
+    with patch.object(
+        mock_bridge1, "hue_activate_scene", return_value=None
+    ) as mock_hue_activate_scene1, patch.object(
+        mock_bridge2, "hue_activate_scene", return_value=None
+    ) as mock_hue_activate_scene2:
+        await hass.services.async_call(
+            "hue",
+            "hue_activate_scene",
+            {"group_name": "group_2", "scene_name": "my_scene"},
+            blocking=True,
+        )
+
+    mock_hue_activate_scene1.assert_called_once()
+    mock_hue_activate_scene2.assert_called_once()
+
+
+async def test_hue_activate_scene_one_responds(
+    hass, mock_bridge1, mock_bridge2, mock_config_entry1, mock_config_entry2
+):
+    """Test that makes only one bridge successfully activate a scene."""
+
+    await setup_component(hass)
+
+    await setup_bridge(hass, mock_bridge1, mock_config_entry1)
+    await setup_bridge(hass, mock_bridge2, mock_config_entry2)
+
+    with patch.object(
+        mock_bridge1, "hue_activate_scene", return_value=None
+    ) as mock_hue_activate_scene1, patch.object(
+        mock_bridge2, "hue_activate_scene", return_value=False
+    ) as mock_hue_activate_scene2:
+        await hass.services.async_call(
+            "hue",
+            "hue_activate_scene",
+            {"group_name": "group_2", "scene_name": "my_scene"},
+            blocking=True,
+        )
+
+    mock_hue_activate_scene1.assert_called_once()
+    mock_hue_activate_scene2.assert_called_once()
+
+
+async def test_hue_activate_scene_zero_responds(
+    hass, mock_bridge1, mock_bridge2, mock_config_entry1, mock_config_entry2
+):
+    """Test that makes no bridge successfully activate a scene."""
+
+    await setup_component(hass)
+
+    await setup_bridge(hass, mock_bridge1, mock_config_entry1)
+    await setup_bridge(hass, mock_bridge2, mock_config_entry2)
+
+    with patch.object(
+        mock_bridge1, "hue_activate_scene", return_value=False
+    ) as mock_hue_activate_scene1, patch.object(
+        mock_bridge2, "hue_activate_scene", return_value=False
+    ) as mock_hue_activate_scene2:
+        await hass.services.async_call(
+            "hue",
+            "hue_activate_scene",
+            {"group_name": "group_2", "scene_name": "my_scene"},
+            blocking=True,
+        )
+
+    # both were retried
+    assert mock_hue_activate_scene1.call_count == 2
+    assert mock_hue_activate_scene2.call_count == 2
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6319846)